README.md

```diff
@@ -123,41 +124,69 @@ Running the ACAP without TLS requires no further setup.
 
 #### TLS Setup
 
-TLS requires a keys and certificates to work, which are listed in the
+TLS requires keys and certificates to work, which are listed in the
 subsections below. For more information on how to generate these files, please
 consult the official [Docker documentation][docker-tls].
-Most of these keys and certificates need to be moved to the Axis device. There are multiple ways to
+Some of these keys and certificates need to be moved to the Axis device. Currently there are multiple ways to
 achieve this, for example by using `scp` to copy the files from a remote machine onto the device.
 This can be done by running the following command on the remote machine:
 
 ```sh
-scp ca.pem server-cert.pem server-key.pem root@<device ip>:/usr/local/packages/dockerdwrapper/
+scp ca.pem server-cert.pem server-key.pem root@$DEVICE_IP:/usr/local/packages/dockerdwrapper/localdata/
 ```
 
 Once copied to the Axis device the correct permissions need to be set for the certificates:
 
 ```sh
-ssh root@<device IP> 'chown acap-dockerdwrapper /usr/local/packages/dockerdwrapper/*.pem'
+ssh root@$DEVICE_IP 'chown acap-dockerdwrapper /usr/local/packages/dockerdwrapper/localdata/*.pem'
 
 ```
 
+Once root ssh access is removed from the device (Axis OS version 12.0 and higher) the preferred method is to use the cert_manager.cgi packaged with the ACAP to install the required keys and certificates on the device with the relevant priviledges. A user with admin rights is required in conjunction with the cert_manager.cgi.
+
+Three HTTP request methods are associated with the cert_manager.cgi, GET, POST and DELETE. GET is used only for accessing help or usage information. POST is used to upload keys or certificates to the device, whereas DELETE is used to remove them. The name of the key or certifcate musr be supplied when POST'ing or DELETE'ing. GET may be used to confirm valid names and example cURL usage. The cert_manager.cgi usage and help can be obtained from your host machine as follows.
+
+```sh
+export DEVICE_IP=<device ip>
+export DEVICE_PASSWORD='<password>'
+
+curl --anyauth --user root:$DEVICE_PASSWORD -X GET "http://$DEVICE_IP/local/dockerdwrapper/cert_manager.cgi"
```

Comment:
should be some other user than root here - do we have some place where different users are described in full (ssh-user, acap-user, ADA(?)-user)?

Reply:
Done